### PR TITLE
Refactor Node and Redis

### PR DESCRIPTION
**Pull request by Busyreadingsomething**

### CODE REVIEW

File: src/nodeServer/models.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+          const menu = JSON.stringify(result.menu[meal]);
+          redisClient.setex(redisKey, 5, menu);
+          response.writeHead(200, { 'Content-Type': 'application/json' });
+          response.end(menu);
+        });
+      }
+    });
+  },
+  filterBy: (request, response, itemInfo) => {
+    // res.set({ 'Access-Control-Allow-Origin': '*' });
```
Comment (MichaelDurfey) [inline src/nodeServer/models.js:44]: You could clear out this comment here

File: src/nodeServer/models.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    response.writeHead(200, { 'Content-Type': 'text/html' });
+    htmlStream.pipe(response);
+  },
+  serveBundle: (request, response) => {
+    const bundleStream = fs.createReadStream('../public/bundle.js');
+    response.writeHead(200, { 'Content-Type': 'text/javascript' });
+    bundleStream.pipe(response);
+  },
+  menuType: (request, response, itemInfo) => {
+    // res.set({ 'Access-Control-Allow-Origin': '*' });
```
Comment (MichaelDurfey) [inline src/nodeServer/models.js:17]: Could also get rid of this

File: src/nodeServer/app.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  serveBundle,
+  menuType,
+  filterBy,
+} = require('./models');
+
+const app = http.createServer((request, response) => {
+  const { method, url } = request;
+
+  if (method === 'GET') {
+    const paths = url.slice(1).split('/');
```
Comment (MichaelDurfey) [inline src/nodeServer/app.js:13]: Why slice this?.. interesting.

**Review by MichaelDurfey — APPROVED** [review]
Looks good. I see you're going back to mongo!

File: src/server/server.js [hunk 4]
```diff
@@ -1,3 +1,4 @@
+require('newrelic');
```
Comment (Angelhendrix) [inline src/server/server.js:1]: do you need two require('newrelic');? 
one on this file and one in the other?

**Pull request merged**